samples/tfm/provisioning_image_net_core/README.rst

```diff
@@ -0,0 +1,33 @@
+.. _provisioning_image_net_core:
+
+Provisioning image for network core
+###################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+
+Running this provisioning image in the network core will disable debugging by enabling the APPROTECT in the UICR register.
```

Comment:
Does it need more explaining when or why you would need to do this (disable debugging for network core), or is it given/clear?

Reply:
In the readme we state that:
"The provisioning images initialize the provisioning process of a device in a manner compatible with Trusted Firmware-M (TF-M)."


The provisioning process of TF-M requires disabling the debugging access. So I hope that it is clear but let me know if you think that we can write it better.